tests/println_multiple_args/println_multiple_args.vv

```diff
@@ -7,5 +7,5 @@ fn main() {
 	for idx, el in arr {
 		println('$idx $el')
 	}
-	println('${strings.repeat('Hello\n', 10)} ${34} ${'hello'}')
+	println('${'Hello\n'.repeat(10)} ${34} ${'hello'}')
```

Comment:
With this change, the `import strings` on line 3 is no longer required.  Compiling tells you
```
tests/println_multiple_args//println_multiple_args.vv:3:8: warning: module 'strings' is imported but never used
    1 | module main
    2 | 
    3 | import strings
      |        ~~~~~~~
    4 | 
    5 | fn main() {
```